RentACar/RentACar/Repositories/Repository.cs

```diff
@@ -0,0 +1,64 @@
+﻿using RentACar.Helpers;
+using RentACar.Intefaces;
+using RentACar.Models.Base;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RentACar.Repositories
+{
+    public abstract class Repository<T> : IRepository<T> where T : Entity
+    {
+        private List<T> _objs;
```

Comment:
let's try to use a better name, I have some ideas but I won't tell you 😛

Reply:
🙄 Hahha.. _list? :P

Reply:
what does this `list` represent?

Reply:
entitys! right? so _entityList ?